src/css/components/Button.scss

```diff
@@ -111,7 +121,7 @@ $Button--negative-bg: $ts-white;
 $Button--negative-bg-hover: $cu-negative;
 $Button--negative-color: $cu-negative;
 $Button--negative-color-hover: $ts-white;
-$Button--negative-border-color: $cu-divider;
+$Button--negative-border-color: #d6d6d6;
```

Comment:
why change to the hard coded value?

Reply:
The negative button was looking funky on some themes, and since the other colors are fixed (we don't theme $cu-negative I don't think, or we shouldn't) I was thinking the negative button would just be fixed...  I guess I was probably viewing it out of context, though, like on a white background instead of a dark background. I can revert this.